v4/source/posts.yaml

```diff
@@ -216,6 +216,42 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/user/{user_id}/posts/{post_id}/set_unread':
+    post:
+      tags:
+        - posts
+      summary: Mark as unread from the post.
+      description: |
+        Mark a channel as being unread from a given post.
+        ##### Permissions
+        Must have `read_channel` permission for the channel the post is in or if the channel is public, have the `read_public_channels` permission for the team.
+        Currently `user_id` should be the same one logged in.
```

Comment:
We're going to have to support the user ID parameter since we're including it in the API route.